sw/device/lib/testing/test_rom/test_rom_start.S

```diff
@@ -187,6 +188,17 @@ _start:
   li   t0, 0x55aa
   sw   t0, EDN_CTRL_REG_OFFSET(a0)
 
+  // Scramble and initialize main memory (main SRAM).
+  // Memory accesses will stall until initialization is complete.
+  // Skip SRAM initialization for DV sim device type, as the testbench handles
+  // this to optimize test run times.
+  lw    t0, kDeviceType
+  beqz  t0, .L_sram_init_skip
```

Comment:
do you think it makes sense to make the comparison to `kDeviceSimDV` so in the future even if the enum values change things will keep working? there's an implicit assumption of 0 here right?

Reply:
Yeah so I wanted to do that, but I can't use the enum value in the assembly. I could assign the enum values to macros and use the preprocessor, but that will change a bit more code. I'm cool with that though, it's not too much more to change.

<img width="1059" alt="Screen Shot 2022-04-01 at 4 07 27 PM" src="https://user-images.githubusercontent.com/5633066/161352832-3ac8c2e3-2903-4909-808c-7ddfb954bfab.png">
.

Reply:
ah naw, in that case i would recommend leaving a gigantic comment in the place where enum is declared. 
Kind of a "thou shalt not change this value"

Reply:
ah ok. will do.

Reply:
fixed.